contracts/SwitcheoTokenZIL.scilla

```diff
@@ -47,30 +53,34 @@ let get_val =
   | None => zero
   end
 
+let some_addr = 
+  fun (addr: ByStr20) =>
+  Some {ByStr20} addr
+
 (***************************************************)
 (*             The contract definition             *)
 (***************************************************)
 
 contract FungibleToken
 (
-  contract_owner: ByStr20,
-  name : String,
+  name: String,
```

Comment:
Will need contract_owner as part of ZRC-2 spec

Reply:
Added!